src/commands/cluster-slot-stats.json

```diff
@@ -0,0 +1,77 @@
+{
+    "SLOT-STATS": {
+        "summary": "Return array of slot usage statistics for slots assigned to the current node",
+        "complexity": "O(N) where N is the total number of slots based on arguments. O(N log N) with ORDERBY subcommand",
+        "group": "cluster",
+        "since": "7.2.0",
+        "arity": -2,
+        "container": "CLUSTER",
+        "function": "clusterSlotStatsCommand",
+        "command_flags": [
+            "STALE"
+        ],
+        "command_tips": [
+            "NONDETERMINISTIC_OUTPUT"
```

Comment:
It's not "RESPONSE_POLICY:AGG_SUM" since we can't just simply aggregate the results. "REQUEST_POLICY:ALL_SHARDS" is probably sensible, I was assuming clients would probably want to merge it natively, but simply doing it by shard isn't that bad.

Reply:
that depends on what's the logic of SUMming a map, if there are duplicate entries we and we sum their values that could be ok.
if not, then we need SPECIAL like in INFO?

Reply:
There shouldn't be duplicate entires, since you are only doing one node from each shard. If there is a duplicate, there is probably a slot migration ongoing and a replica has a slot owned by another node. We would want to just pick one.

Reply:
Should we have a response policy of "pick one"?

Reply:
ohh, right. for some reason i thought about merging slots into one report, but that defeats the purpose of that command..
so i guess just add ALL_SHARDS